### ISSUE TITLE
must register native routines (new winbuilder NOTE)

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1209]
The info I get from winbuilder is as follows:
```
* checking compiled code ... NOTE
File 'oce/libs/i386/oce.dll':
  Found no calls to: 'R_registerRoutines', 'R_useDynamicSymbols'
File 'oce/libs/x64/oce.dll':
  Found no calls to: 'R_registerRoutines', 'R_useDynamicSymbols'

It is good practice to register native routines and to disable symbol
search.

See 'Writing portable packages' in the 'Writing R Extensions' manual.
```

**Comment by dankelley** [comment]
A bit of web searching tells me that we are supposed to follow directions at https://cran.r-project.org/doc/manuals/r-devel/R-exts.html#Registering-native-routines but they are really quite complex, so instead I'm going to start by mimicking the checked answer at http://stackoverflow.com/questions/42313373/r-cmd-check-note-found-no-calls-to-r-registerroutines-r-usedynamicsymbols.

**Comment by dankelley** [comment]
NOTE: this is for `r-devel`.

**Comment by dankelley** [comment]
Fixed in `develop` commit f84f1ab97fcefeca4844c0d76e50cefbdd6b46f2 (works on winbuilder R-release and R-devel, and on local osx tests with R 3.3.3)
